### PR TITLE
[FEATURE] added fixture to data sets, closes #9

### LINKED ISSUE #9: Add fixture to training set
- possibly increases accuracy of "position" input parameter evaluation

### PR DESCRIPTION
**Pull request by PiMastah**
- added getLastPositionForTeam based on new view position_by_team_and_date (returns only position)
- changed parsing of future matches to **not** set position to 1 (needs testing with regards to behaviour for first matchday in a season)
- added positional information to upcoming matches missing this information similar to market values for upcoming matches, will use the last available position which is usually from the next upcoming match (after a matchday is complete, upcoming matches have information on the position already)
- input layer size of the neural network is now determined by the size of the input from the view
- changed network architecture (needs further fine tuning in issue #4 )

### CODE REVIEW

**Comment by lianmakesthings** [comment]
LGTM
